tests/Fixtures/openapi.yaml

```diff
@@ -176,6 +176,54 @@ paths:
                                 example: ""
             tags:
                 - 'Group A'
+    '/api/withExampleTagTypeResponse':
+        get:
+            summary: ''
+            operationId: getApiWithExampleTagTypeResponse
+            description: ''
+            parameters:
+                -
+                    in: header
+                    name: Custom-Header
+                    description: ''
+                    example: NotSoCustom
+                    schema:
+                        type: string
+            responses:
+                200:
+                    description: ''
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                example: ''
+            tags:
+                - 'Group A'
+            security: []
+    '/api/withExampleTagTypeRequest':
+        get:
```

Comment:
Hmmm, doesn't seem like the contents match what should be generated here. I don't see any example requests.